pom.xml

```diff
@@ -273,6 +295,12 @@
                 <artifactId>util-testing</artifactId>
                 <version>1.17.0-SNAPSHOT</version>
             </dependency>
+            <dependency>
+                <groupId>com.github.krallin</groupId>
+                <artifactId>tini</artifactId>
+                <version>${tini.version}</version>
```

Comment:
Where is this property defined?

Reply:
The tini.version is not explicitly defined in develop branch also. 
It looks like, the version is referred from its internal configuration files. Please refer the tini project on below link. 
https://github.com/krallin/tini
I will also try to find out exactly from where it is referring to the tini.version.

Reply:
@dermot-hardy , 
The commit https://github.com/CAFapi/caf-common/commit/7f73df08f37de53f9e7e36b3aaf67d0c3a1af190 has the change for removing tini.version. Can we add the tini.version back to parent pom?

Reply:
@michael-bryson , As Dermot suggested tini.version property with value 0.16.1 added.

Reply:
Why are we adding a property when this is only used in one place? We should just add the version instead of the token and remove the property.